.github/workflows/testing.yml

```diff
@@ -33,26 +33,29 @@ jobs:
           node-version-file: .nvmrc
           cache: npm
 
-      - name: Install latest npm
-        # TODO: npm 10 requires Node.js 18.17 or later.
-        # run: npm install --global npm@latest
-        run: npm install --global npm@9
-
       - name: Install dependencies
         run: npm ci
 
-      - name: Install stylelint v15
-        run: npm i -D stylelint@^15
-        if: matrix.stylelint == 15
+      - name: Install Stylelint 16
+        run: npm i -D stylelint@16 stylelint-scss@6
+        if: ${{ matrix.stylelint == '16' }}
+
+      - name: Install Stylelint 15
+        run: npm i -D stylelint@15 stylelint-scss@5
+        if: ${{ matrix.stylelint == '15' }}
 
-      - name: Install stylelint v14
-        run: npm i -D stylelint@^14 stylelint-scss@^4.3.0
-        if: matrix.stylelint == 14
+      - name: Install Stylelint 14
+        run: npm i -D stylelint@14 stylelint-scss@4.3
+        if: ${{ matrix.stylelint == '14' }}
 
       - name: Run unit tests
+        env:
+          NODE_OPTIONS: --experimental-vm-modules
```

Comment:
[suggestion] `npm run test:unit` without `--experimental-vm-modules` passes on my local environment. Does this command need it? Or can we remove it?

```suggestion
```